core/src/main/java/io/parsingdata/metal/token/IterableToken.java

```diff
@@ -0,0 +1,64 @@
+package io.parsingdata.metal.token;
```

Comment:
Add the license above this new file (see other source files).

Reply:
Ok, done! :)